<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JetLag</class>
 <widget class="QMainWindow" name="JetLag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JetLag</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cboOriginName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cboDestinationName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdGetPlan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Sleep Suggestion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOrigin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin City</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDestination">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination City</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblDepartureHour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hour of Departure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sleep Suggestion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flight Distance (KM)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Travel Time (Hours)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrival Hour:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtFlightDistance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtTravelTime">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtArrivalHour">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtSleepSuggestion">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;happy&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
